cmd/release-controller/http_helper.go

```diff
@@ -792,6 +792,16 @@ func (s newestSemVerFromSummaries) Less(i, j int) bool {
 	if c < 0 {
 		return false
 	}
+	vi, erri := semverParseTolerant(s.summaries[i].From)
```

Comment:
This is expensive to do on every compare -  the point of newestSemVerFromSummaries is to bypass the need to generate these.  If you're going to sort on them, add a new `fromVersions []SemVer` array and init it on creation, same logic.